gnd/src/main/java/com/google/android/gnd/service/firestore/FirestoreDatastore.java

```diff
@@ -41,16 +41,16 @@
 import javax.inject.Singleton;
 
 @Singleton
-public class FirestoreDataService implements RemoteDataService {
+public class FirestoreDatastore implements RemoteDatastore {
```

Comment:
(And this would be `FirestoreDataStore`, since "Firestore" is a one word product name and "data store" is a two word generic phrase...)

Reply:
Done.